### PR TITLE
Fix for issue #785; readme formatting

### PR DESCRIPTION
**Pull request by mbudiu-vmw**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -1,302 +0,0 @@
-# Copyright 2013-present Barefoot Networks, Inc.
```
Comment (cc10512) [inline Makefile.am]: I don't understand why do you need to remove Makefile.am again. Have you rebased?

Comment (mbudiu-vmw) [inline Makefile.am]: I have, this is built with cmake.
I don't know why the files are still there.

Comment (mbudiu-vmw) [inline Makefile.am]: I guess maybe not this particular branch...

Comment (cc10512) [inline Makefile.am]: git likely does not remove your local files once you've update the branch. You can run `git clean -dxf` after you rebased your master.

Removing them from .gitignore is a good catch!

File: ir/node.cpp [hunk 2]
```diff
@@ -102,6 +102,17 @@ Util::JsonObject* IR::Node::sourceInfoJsonObj() const {
     return json;
 }
 
+static cstring quote(cstring s) {
```
Comment (cc10512) [inline ir/node.cpp]: shouldn't this be a method of cstring?

Comment (mbudiu-vmw) [inline ir/node.cpp]: This quoting is really specific to JSON.
For C you would need more cases.
So I decided to make it local for now.

Comment (mbudiu-vmw) [inline ir/node.cpp]: I could move this method to cstring and call it jsonQuote.

Comment (cc10512) [inline ir/node.cpp]: would be nice, because otherwise, nobody is going to find it and it will be bound to repeat. A comment on it would also help :)

**Pull request merged**